R-packages/evalcast/R/plot_trajectory.R

```diff
@@ -97,7 +98,22 @@ plot_trajectory <- function(list_of_predictions_cards,
   response_df <- response_df %>% dplyr::mutate(
     location_abbr = if(geo_type == "state") covidcast::fips_to_abbr(paste0(location,"000")) else covidcast::fips_to_name(location)) %>%
     dplyr::filter(!is.na(location_abbr))
-  ggplot(preds_df, aes(x = .data$reference_period)) +
+    
+    #set the page number 
+  fc_round <- function(x){  #round to integer
+    idx = rep(c(FALSE,TRUE), length.out=length(x))
+    floor(x)*idx + ceiling(x)*(!idx)
+  }
+  
+  if (length(unique(preds_df$location_abbr))/45 < 1) {
+    page_facet = page_facet
+  } else 
+    page_facet = fc_round(length(unique(preds_df$location_abbr))/45)
+  
```

Comment:
Here, we do
```
nfacets = length(unique(preds_df$location_abbr))
if(ncol*nrow*page_facet < nfacets) page_facet = ceiling(nfacets / (ncol*nrow))
```
We have to figure out how to let one or both of ncol or nrow be `NULL` so we can just let `facet_wrap()` do it's thing in those cases.